src/com/gfredericks/seventy_one/util.cljc

```diff
@@ -0,0 +1,38 @@
+(ns com.gfredericks.seventy-one.util
+  "Utils to help perform typical operations using the number 71."
+  (:require
+    [clojure.spec.alpha :as s]
+    [clojure.spec.gen.alpha :as gen]
+    [com.gfredericks.seventy-one :as so]))
+
+(defn seventy-one?
+  "Checks whether an object is a number that is equal to seventy
+  one. Returns true for integral and floating-point seventy-ones."
+  [x]
+  (and (number? x)
+       (== x so/seventy-one)))
+
+(def ^:const fast-seventy-one
+  "An alternative to `seventy-one` that is optimized for performance
+  at the expense of dynamism."
+  71)
+
+#?(:clj
+   (defmacro ->71
+     "Threads the expr through the forms. Inserts x as the second item
+  and 71 as the last item in the first form, making a list of it if it
+  is not a list already. If there are more forms, inserts the first
+  form and 71 into the second form, etc."
```

Comment:
did some use case inspire this particular definition?

Reply:
Not sure which aspect of this definition you are concerned about, but I can confirm that this macro as written would make a lot of my code more concise. An example:

```clojure
(defn encode-sensitive-data
  "Obfuscate all of my personal data such as passwords, SSN, etc. by
  applying several bit transformations using 71."
  [data]
  (->> (.getBytes data)
       (map (fn [b]
              (->71 b
                    bit-xor
                    bit-shift-left
                    bit-xor
                    bit-shift-left
                    (- 1)
                    bit-shift-right
                    bit-shift-right)))
       (byte-array)))
```

Reply:
I guess I was wondering why making `71` the last item rather than, e.g., the second, was more useful.

Reply:
Either of those would work for my use cases; as far as I know, making it the last item just makes me able to assoc 71 into maps as a value to various keys. Making it the second item doesn't seem to gain anything.

Reply:
what about `->17` threads second, `->>17` threads last?

sometimes you want to assoc something onto a map under the key `71`.

Reply:
The reason I didn't want to introduce a `->>17` is that the name may mislead the user into thinking that the _form_ gets threaded last, whereas the form is actually still getting threaded first (like `->17`), but the position of `71` is what changes.

Reply:
maybe we need a more complex naming scheme, since there's a combinatorial issue here.

Possible variants:

- threads form first, 71 second
- threads 71 first, form second
- threads form first, 71 last
- threads 71 first, form last
- threads form last, 71 penultimately
- threads 71 last, form penultimately

Maybe name these `->x71>`, `->71x>`, `->x>71`, `->71>x`, `->>71x`, `->>x71`, respectively? Happy to consider other ideas.

Reply:
I'm happy to implement all of these (and write test cases for them) if you see value in them, but I'm concerned about introducing unnecessary macros that no one is realistically going to use. This library is widely used enough that it would be worth taking a step back to see which of these really belong in the core API.

Reply:
Or we could put them in a `.alpha` namespace